fmt/tests/basics/buildfile

```diff
@@ -1,9 +1,13 @@
 import! [metadata, rule_hint=cxx.link] libs = fmt%lib{fmt}
 
-./ : exe{driver} : {cxx}{driver} hxx{tests.inl} $libs testscript{**}
+./ :
```

Comment:
Is that  line still necessary?

Reply:
Apparently not ;)
I may have thought it more clear since the following two are both conditional, but indeed seems superfluous.